translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"of file basenames on disk\n"
+"\t\t  kubectl create configmap my-config --from-file=key1=/path/to/bar/file1."
+"txt --from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Create a new configmap named my-config with key1=config1 and "
+"key2=config2\n"
+"\t\t  kubectl create configmap my-config --from-literal=key1=config1 --from-"
+"literal=key2=config2"
+msgstr ""
+"\n"
+"\t\t  # Criar um novo configmap com o nome de my-config baseado na pasta "
+"bar\n"
+"\t\t  kubectl create configmap my-config —from-file=path/to/bar\n"
+"\n"
+"\t\t  # Cria um novo configmap com o nome de my-config especificando as "
+"chaves ao invés de utilizar o nome do arquivo em disco\n"
+"\t\t  kubectl create configmap my-config —from-file=key1=/path/to/bar/file1."
+"txt —from-file=key2=/path/to/bar/file2.txt\n"
+"\n"
+"\t\t  # Criar um novo configmap com o nome de my-config com key1=config1 e "
+"key2=config2\n"
+"\t\t  kubectl create configmap my-config —from-literal=key1=config1 —from-"
+"literal=key2=config2"
+
+#: pkg/kubectl/cmd/create_secret.go:135
+msgid ""
+"\n"
+"\t\t  # If you don't already have a .dockercfg file, you can create a "
+"dockercfg secret directly by using:\n"
+"\t\t  kubectl create secret docker-registry my-secret --docker-"
+"server=DOCKER_REGISTRY_SERVER --docker-username=DOCKER_USER --docker-"
+"password=DOCKER_PASSWORD --docker-email=DOCKER_EMAIL"
+msgstr ""
+"\n"
+"\t\t  # Se você ainda não tem o arquivo .dockercfg, você pode gerar "
+"diretamente o dockercfg secret utilizando o comando:\n"
+"\t\t  kubectl create secret docker-registry my-secret —docker-"
+"server=DOCKER_REGISTRY_SERVER —docker-username=DOCKER_USER —docker-"
+"password=DOCKER_PASSWORD —docker-email=DOCKER_EMAIL"
+
+#: pkg/kubectl/cmd/top_node.go:65
+msgid ""
+"\n"
+"\t\t  # Show metrics for all nodes\n"
+"\t\t  kubectl top node\n"
+"\n"
+"\t\t  # Show metrics for a given node\n"
+"\t\t  kubectl top node NODE_NAME"
+msgstr ""
+"\n"
+"\t\t  # Mostra as métricas para todos os nodes\n"
+"\t\t  kubectl top node\n"
+"\n"
+"\t\t  # Mostra as métricas para um node específico\n"
+"\t\t  kubectl top node NODE_NAME"
+
+#: pkg/kubectl/cmd/apply.go:84
+msgid ""
+"\n"
+"\t\t# Apply the configuration in pod.json to a pod.\n"
+"\t\tkubectl apply -f ./pod.json\n"
+"\n"
+"\t\t# Apply the JSON passed into stdin to a pod.\n"
+"\t\tcat pod.json | kubectl apply -f -\n"
+"\n"
+"\t\t# Note: --prune is still in Alpha\n"
+"\t\t# Apply the configuration in manifest.yaml that matches label app=nginx "
+"and delete all the other resources that are not in the file and match label "
+"app=nginx.\n"
+"\t\tkubectl apply --prune -f manifest.yaml -l app=nginx\n"
+"\n"
+"\t\t# Apply the configuration in manifest.yaml and delete all the other "
+"configmaps that are not in the file.\n"
+"\t\tkubectl apply --prune -f manifest.yaml --all --prune-whitelist=core/v1/"
+"ConfigMap"
+msgstr ""
+"\n"
+"\t\t# Aplica a configuração do pod.json a um pod.\n"
```

Comment:
I would insert a "do arquivo pod.json" to get a better context here. Like "Aplica a configuração do arquivo pod.json a um pod"